### DISCUSSION TITLE
機器人能像哆啦A夢吃人類食物嗎？日本機器人博士回答網友問題！

### DISCUSSION THREAD

**Discussion by RogelioKG** [1101-GS3073-C/Python-Community-GS3073-C#642] [General]
Content:
https://youtu.be/VgEG19zGPUs

Reflections / Thoughts:
單純是一個酷酷的影片，分享個，
裡免包含了一些關於 AI & Robotics 的神奇問題，
博士會很滑稽但又一本正經地回答問題 (？)
最神奇的是有人問：機器人能不能吃人類食物，
攝取食物獲得能量對於機器人應該是相當低效的方法才對，
不過博士給出了一個特別的觀點，我覺得見解很獨到。
他認為用餐是一種人類的溝通方式，
那麼讓機器人加入這項能和人類一起用餐的功能，
其目的應該是超越於攝取能量的必要性，
例如是為了達成聊天、陪伴之類的目的存在。

**Comment by larry40305** [comment]
現在糧食都不一定夠給人類吃了，如果還要給機器人吃會夠嗎。

**Comment by bww0401** [comment]
我認為讓機器人跟人類一起用餐的功能，能夠多少幫助獨居老人有溝通的管道，如果再加上看護的功能，可以有效解決現在社福人員不足的問題

**Comment by Aerofe** [comment]
我認為讓機器人飲食是不必要的，食物並不能提供機器人動力來源，還不如把糧食留給人類

**Comment by RogelioKG** [comment]
「目的應該是超越於攝取能量的必要性」
這很清楚，如果這項功能加入，
那它並不是為了達成攝取能量的目的加入的。
嘿，各位大大，我認可糧食在現今的世界是非常缺乏的物資，但我認為這篇討論應該著重在機器人加入飲食功能的原因，而不是糧食究竟缺不缺乏之類的打稻草人謬誤，還請各位大大必須明白這點。
